app/models/notification.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "extensions/symbolize_json"
+
+class Notification < ApplicationRecord
+  include SymbolizeJSON
+
+  def subscriber_lists
+    subscriber_list_query.lists
+  end
+
+private
+
+  def subscriber_list_query
```

Comment:
I'm not sure this should be the concern of the `Notification` object. Obvs, we haven't really settled on the architecture of the matching but I would imagine it to happen outside of here.